databuilder/models/dashboard_metadata.py

```diff
@@ -24,23 +27,27 @@ class DashboardMetadata(Neo4jCsvSerializable):
     Lastreloadtime is the time when the Dashboard was last reloaded.
     """
     DASHBOARD_NODE_LABEL = 'Dashboard'
-    DASHBOARD_KEY_FORMAT = '{dashboard_group}://{dashboard_name}'
+    DASHBOARD_KEY_FORMAT = '{product}_dashboard://{cluster}.{dashboard_group}/{dashboard_name}'
```

Comment:
@jhettler Neo4j key on Dashboard, Dashboard group, Dashboard description is changing which can be backward compatible -- I think it's limited to you as there's no consumer in OSS codebase.

Main reason of changing is to add more fields to avoid collision. The other reason is to fix potential bug where key is defined by `dashboard group name` and `dashboard name` where the key could be not a unique value depends on the Dashboard vendor (Mode allows duplicate names)

Let me know if you have any concern.